modules/admin-ui-interface/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>opencast-admin-ui-interface</artifactId>
+    <packaging>bundle</packaging>
+    <name>Opencast :: admin-ui-interface</name>
+    <parent>
+        <groupId>org.opencastproject</groupId>
+        <artifactId>base</artifactId>
+        <version>14-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <properties>
+        <opencast.basedir>${project.basedir}/../..</opencast.basedir>
+        <checkstyle.skip>false</checkstyle.skip>
+        <interface.url>https://github.com/opencast/opencast-admin-interface/releases/download/2023-04-12-2/oc-admin-ui-2023-04-12.tar.gz</interface.url>
```

Comment:
Pretty much, yeah.

Arguably it would be better to have a branch and release structure more similar to Opencast, so you could more easily backport changes, but for now I went with a way I know how to set up ^^'

Reply:
That's fair.  There's a conversation to be had in the community in terms of how this *should* work for all three modules.